__tests__/app.test.js

```diff
@@ -0,0 +1,95 @@
+const app = require("../app");
+
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+
+afterAll(() => db.end());
+
+// describe("app", () => {

```

Comment:
Please remove any comments, old code or console.logs you are no longer using. We want our code to be readable and clean